### PR TITLE
I added my introduction as a part of HW 1.

### PR DESCRIPTION
**Pull request by soaham-bharti**
Soaham Bharti. I had to add another commit/pull request because I had missed some parts of the grading rubric the first time around. Not too challenging, but my GitHub interface was somewhat different from what was shown in Professor Soltoff's video.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,12 @@
...
+## My introduction
+
+My name is **Soaham Bharti**. I am a first year PhD student in behavioral marketing at the *Booth School of Business*.
+
+Thing I want to study:
+
+* How consumers and other decision makers navigate uncertainty
+* People's appraisal of algorithms and other forecasting methods
+
+Here is a [link](https://research.chicagobooth.edu/cdr/phd-students/soaham-bharti) to my Booth profile page.
```
Comment (ally-bauer) [inline README.md:12]: The link works great, and the page is cool!  It's awesome that you already have a publication to list.

File: README.md [hunk 2]
```diff
@@ -1 +1,12 @@
 # hw01
+
+## My introduction
+
+My name is **Soaham Bharti**. I am a first year PhD student in behavioral marketing at the *Booth School of Business*.
+
+Thing I want to study:
+
+* How consumers and other decision makers navigate uncertainty
+* People's appraisal of algorithms and other forecasting methods
```
Comment (ally-bauer) [inline README.md:10]: A good list with interesting information.  Good work!

File: README.md [hunk 3]
```diff
@@ -1 +1,12 @@
 # hw01
+
+## My introduction
+
+My name is **Soaham Bharti**. I am a first year PhD student in behavioral marketing at the *Booth School of Business*.
```
Comment (ally-bauer) [inline README.md:5]: Good job on the coding!

Comment (lindsaykkk) [inline README.md:5]: A good use of italics and bold in one sentence!

File: README.md [hunk 4]
```diff
@@ -1 +1,12 @@
 # hw01
+
+## My introduction
```
Comment (lindsaykkk) [inline README.md:3]: Overall, it's a pretty concise but informational bio that has utilized RMarkdown well!

File: README.md (hunk 1 above)
Comment (lindsaykkk) [inline README.md:12]: The link is great, and maybe you could consider inserting a picture the next time. Overall, good job!

File: README.md [hunk 5]
```diff
@@ -1 +1,12 @@
...
+
+My name is **Soaham Bharti**. I am a first year PhD student in behavioral marketing at the *Booth School of Business*.
+
+Thing I want to study:
+
+* How consumers and other decision makers navigate uncertainty
```
Comment (Donghyun-Kang-Soc) [inline README.md:9]: Try different combinations of lists (either unordered and ordered list or two different levels of unordered lists) next time.

**Review by Donghyun-Kang-Soc — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Improvement |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |     X         |            |
| **Coding strategy**      |           |      X        |            |
| **Presentation**            |           |       X       |           |
| **Achievement, creativity** |           |      X        |            |
| **Reproducibility**         |     X      |             |            |

Remarks:
* You used headers, emphasis, an unordered list, and a link, which hit the four aspects of the markdown. But please try to experiment with other elements as the course proceed when you submit other assignments.
* I expected to see a bit more of your biography and some combinations of different markdown aspects.

**Pull request closed without merging**